node/network/availability-distribution/src/error.rs

```diff
@@ -104,7 +108,8 @@ pub fn log_error(result: Result<()>, ctx: &'static str) -> std::result::Result<(
 				JfyiError::FetchPoV(_) |
 				JfyiError::SendResponse |
 				JfyiError::NoSuchPoV |
-				JfyiError::Runtime(_) => gum::debug!(target: LOG_TARGET, error = ?jfyi, ctx),
+				JfyiError::Runtime(_) =>
+					gum::warn_if_frequent!(freq: warn_freq, max_rate: gum::MAX_FREQ_RATE, target: LOG_TARGET, error = ?jfyi, ctx),
```

Comment:
nit: I would rename `MAX_FREQ_RATE`, to something that has units in the name `ONE_SECOND_FREQ_RATE` this way a future reader/user of this API wouldn't have to read the implementation to understand how to use this macro.